rest_framework/relations.py

```diff
@@ -62,10 +71,12 @@ def __init__(self, **kwargs):
         self.queryset = kwargs.pop('queryset', self.queryset)
         self.html_cutoff = kwargs.pop('html_cutoff', self.html_cutoff)
         self.html_cutoff_text = kwargs.pop('html_cutoff_text', self.html_cutoff_text)
-        assert self.queryset is not None or kwargs.get('read_only', None), (
-            'Relational field must provide a `queryset` argument, '
-            'or set read_only=`True`.'
-        )
+
+        if not method_overridden('get_queryset', RelatedField, self):
+            assert self.queryset is not None or kwargs.get('read_only', None), (
+                'Relational field must provide a `queryset` argument, '
+                'or set read_only=`True`.'
```

Comment:
Should this now read "Relational field must provide a `queryset` argument, override `get_queryset`, 'or set read_only=`True`."?